<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1066</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTreeView" name="samples_tree">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QGraphicsView" name="camera_tl"/>
      </item>
      <item row="0" column="2">
       <widget class="QGraphicsView" name="camera_tr"/>
      </item>
      <item row="1" column="0">
       <widget class="QGraphicsView" name="camera_bl"/>
      </item>
      <item row="1" column="2">
       <widget class="QGraphicsView" name="camera_br"/>
      </item>
      <item row="1" column="4">
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QSlider" name="timeline">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="prev_frame">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>⏴⏴</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="play_pause">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="next_frame">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>⏵⏵</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="4">
       <widget class="SkeletonView" name="skeletal" native="true"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1066</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_samples"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_samples">
   <property name="text">
    <string>Load samples</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SkeletonView</class>
   <extends>QWidget</extends>
   <header>gui/skeleton_view.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionOpen_samples</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openDirectory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>exitApp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>samples_tree</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>openSequence(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>prev_frame</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>prevFrame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>834</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>next_frame</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>nextFrame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1006</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>timeline</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>setFrame(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>920</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>nextFrame()</slot>
  <slot>prevFrame()</slot>
  <slot>playPause()</slot>
  <slot>exitApp()</slot>
  <slot>openDirectory()</slot>
  <slot>openSequence(QModelIndex)</slot>
  <slot>setFrame(int)</slot>
 </slots>
</ui>
